src/main/java/graphql/nadel/schema/SchemaTransformationHook.java

```diff
@@ -0,0 +1,46 @@
+package graphql.nadel.schema;
+
+import graphql.PublicSpi;
+import graphql.schema.GraphQLSchema;
+
+/**
+ * High level representation of a a graphql schema transformation. Warning: the schema resulting from the transformation
```

Comment:
Maybe make clear that this is actually the overall schema that is transformed, not the underlying one.

Reply:
Done.